external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -136,9 +137,48 @@ class KafkaTestUtils(
     kdcConf.setProperty(MiniKdc.DEBUG, "true")
     kdc = new MiniKdc(kdcConf, kdcDir)
```

Comment:
Hi, @AngersZhuuuu . Maybe, I missed the discussion history.
I'm wondering if we can put the config to `kdcConf` before creating `MiniKdc`?

Reply:
> Hi, @AngersZhuuuu . Maybe, I missed the discussion history.
> I'm wondering if we can put the config to `kdcConf` before creating `MiniKdc`?

Now we use hadoop's MiniKDC, `krb5.conf` generated in MiniKDC.
If we want to put krb5conf before creating `MiniKDC`, we should build our own `MiniKDC`
https://github.com/apache/spark/pull/26594#discussion_r350561381

Reply:
Got it.